flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/sink/TestSinkV2.java

```diff
@@ -241,7 +242,8 @@ public TestSinkV2WithPostCommitTopology(
 
         @Override
         public void addPostCommitTopology(DataStream<CommittableMessage<String>> committables) {
-            // We do not need to do anything for tests
+            StandardSinkTopologies.addGlobalCommitter(
```

Comment:
Nit: Should we open a follow-up to increase the coverage even further by also covering the expansions of pre/post-commit topology?

Reply:
That's a good point. I created https://issues.apache.org/jira/browse/FLINK-36807 .